obstacle_check_angle: 0.04
obstacle_range_threshold: 2.5
branch_range_max: 6
branch_range_min: 0.3
branch_diff_x_min: 1
branch_diff_x_max: 8
branch_diff_y_min: 0
branch_diff_y_max: 10
scan_filter: false
scan_filter_order: 3
branch_filter: false
branch_filter_order: 3
branch_filter_tolerance: 0.2
duplicate_detection: true
duplicate_tolerance: 0.8
log_current_time: 10
newer_duplication_threshold: 120
on_map_branch_detection: true
omb_map_window_x: 1
omb_map_window_y: 1
on_map_branch_rate: 0.5
